src/rules/braceStyleRule.ts

```diff
@@ -148,4 +161,17 @@ class BraceStyleWalker extends Lint.RuleWalker {
   private getEndPosition(node: ts.Node) {
     return node.getSourceFile().getLineAndCharacterOfPosition(node.getEnd());
   }
+
+  // returns previous node which is either block or catch clause (no keywords, etc).
+  private getPreviousNode(children: ts.Node[], node: ts.Node): ts.Node {
+    let position = children.indexOf(node) - 1;
+    while (position >= 0) { // is first child always block or catch clause?
```

Comment:
hi @hadalin, for readability purposes would you mind refactoring this method so that there are no variable declarations inside the loop? You can also get rid of the `else` block and simply decrease the `position`.